src/Email.php

```diff
@@ -101,31 +99,18 @@ private function replaceBodyText(): void
                     $toReplace = $m;
                     $m = str_replace('[[', '', str_replace(']]', '', $m));
                     $parts = explode('.', $m);
+
+                    $replacementString = $this->replacements[$toReplace];
                     
-                    // Here the first part will be the Model name, whereas
-                    // the second is the field to replace with.
-                    
-                    // Create a new instance of the required class.
-                    $modelClass = Str::studly(Str::singular($parts[0]));
-                    $modelName = '\\App\\Models\\' . $modelClass;
-                    $model = null;
-
-                    switch($modelClass) {
-                        case 'User':
-                        case 'Issuer':
-                            $model = $modelName::find($this->modelId);
-                            break;
-                        case 'Organisation':
-                            $model = $modelName::find($this->byId);
-                            break;
-                        default:
-                            // Unknown model
-                            break;
-                    }
-
-                    // Now replace the placeholder within the body with the
-                    // required model field.
-                    $replacementString = $model->{$parts[1]};
+                    try {
+                        //Get the model class by string name
+                        $modelClass = Str::studly(Str::singular($parts[0]));
+                        $modelName = '\\App\\Models\\' . $modelClass;
+
+                        $model = $modelName::find($this->modelId);
+
+                        $replacementString = $model->{$parts[1]};
+                    } catch(e) {}
```

Comment:
We should throw here, at the very least. Otherwise it fails silently, but to calling code, looks like it completed.